### ISSUE TITLE
RX580 issue with Legacy Miner

### ISSUE THREAD

**Issue by prosoftcorp** [nicehash/NiceHashMiner-Archived#1523]
Hello, 
I have reinstalled windows several times my AMD RX580 not being detected in NHML or it crashes Tried with: Windows 10 Pro and Windows 7 Pro both 64 Bit Versions Tried with: AMD Drivers Starting from 17.7.1 and sever others including latest drivers. Kindly assist with the combination to use, shall use 2 rigs with 8 GPU each. 

Rgds Sandeep

**Comment by Umair-khalidkhan** [comment]
Same was a problem with me . I updated all drivers and this happened( with 3 RX 570s). I then reinstalled the windows and then used the drivers that came with the GPU as they were previously working. This worked for me though.

**Comment by prosoftcorp** [comment]
Dear Mr. Umair

 

Can you pls tell me which windows version you used, as my cards are RX580
Nitro+ 8GB

 

Thanks and regards

Sandeep

 

  _____

**Comment by Umair-khalidkhan** [comment]
I had windows 10, 64 bit
Plus nicehash Is very unaccomodating. Thinking of switching to ethereum

**Comment by prosoftcorp** [comment]
Dear Mr. Umair

 

I have installed SimpleMiningOS and using Nanopool now, but for the sake of
simplicity and account keeping I wanted to use NiceHash as all my A9 ZMaster
machines are on NiceHash currently. 

 

I am not operating the GPU rigs as I don't get any proper exposure on the
Nanopool Dashboard, I also want to know what to mine on these 2 Rigs with 8
GPU each. Tha'ts whay I waned to use windows with a simple Nicehash
interface.

 

Best regards

Sandeep

 

  _____

**Comment by Umair-khalidkhan** [comment]
Yes nicehash is very easy to use. However their customer care is very poor. Has my solution done any good?
